src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -68,7 +68,7 @@ public void processDailyList(DailyListEntity dailyListEntity) throws JsonProcess
                         String caseNumber = getCaseNumber(dailyListEntity, dailyListHearing);
 
                         HearingEntity hearing = retrieveCoreObjectService.retrieveOrCreateHearing(
-                            courtHouseName, String.valueOf(sitting.getCourtRoomNumber()),
```

Comment:
Could that Integer come through as a null?

Reply:
Shouldn't do as it's required in the XSD.